src/pages/profile-page/profile-page.js

```diff
@@ -0,0 +1,312 @@
+                    <Button component='span' className={classes.uploadBtn}>
+                      {images
+                        ? images.thumbnail
+                          ? PROFILE_DATA[language].changePhoto
+                          : PROFILE_DATA[language].addPhoto
+                        : PROFILE_DATA[language].addPhoto}
+                    </Button>
+                  </label>
+                </div>
+                <div className={classes.userNames}>
+                  <TextField
+                    label={PROFILE_DATA[language].firstName}
+                    name='firstName'
+                    value={user.firstName || ''}
+                    className={classes.userInput}
+                    onChange={handleChange}
+                  />
+                  <TextField
+                    label={PROFILE_DATA[language].lastName}
+                    name='lastName'
+                    value={user.lastName || ''}
+                    className={classes.userInput}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className={classes.restOfUserInputs}>
+                <TextField
+                  label={PROFILE_DATA[language].email}
+                  name='email'
+                  value={user.email || ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].phoneNumber}
+                  name='phoneNumber'
+                  value={user.phoneNumber ? user.phoneNumber : ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].country}
+                  name='country'
+                  value={
+                    user.address
+                      ? user.address.country
+                        ? user.address.country
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].region}
+                  name='region'
+                  value={
+                    user.address
+                      ? user.address.region
+                        ? user.address.region
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].city}
+                  name='city'
+                  value={
+                    user.address
+                      ? user.address.city
```

Comment:
Why do you have `user.address.city` twice here?

Reply:
removed